### ISSUE TITLE
Very high CPU usage on tooling-force.jar

### ISSUE THREAD

**Issue by rbarna1** [ChuckJonas/vscode-apex-autocomplete#26]
I took the latest updates to VSCode yesterday (July 31) and ever since, the Salesforce Language Support plugin Java process has been taking 100% CPU on my Win 10 machine.  Is there a known reason for this?
I notice as well that this Java process does not kill when closing down VSCode (screenshot attached)

![image](https://user-images.githubusercontent.com/821154/28846794-b89313fc-76da-11e7-90ad-6c069f12bdc2.png)

**Comment by ChuckJonas** [comment]
Ya, I've been experiencing similar issues in recent updates (actually going back into May).  Does the auto-complete still respond?  

I've been meaning to dig into this, but haven't been able to find the time.

**Comment by rbarna1** [comment]
Yeah, I think it does.  I've been having to kill the process lately, because, well, 100% CPU is too much to give.  But while running I think it has been working.

**Comment by ChuckJonas** [comment]
how many apex classes do you have in your project?

also, you could try increasing the latency on `apexAutoComplete.checkSyntaxDelay` (up to about `1500` the delay isn't horrible) and see if that helps at all.
